subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/plugins/AbstractPluginBuildIntegrationTest.groovy

```diff
@@ -41,6 +45,11 @@ abstract class AbstractPluginBuildIntegrationTest extends AbstractIntegrationSpe
         final TestFile projectPluginFile
 
         PluginBuildFixture(String buildName, boolean useKotlinDSL) {
+            this(null, buildName, useKotlinDSL)
+        }
+
+        PluginBuildFixture(String rootDir, String buildName, boolean useKotlinDSL) {
```

Comment:
I don't think this is a good idea.  We have other fixtures that do something like this and the String concatenation makes it hard to follow what paths are being used. 

Maybe something like `TestFile rootDir` could be used here. But we'd need to make other changes:
1. PluginBuildFixture would be made static
2. The default for rootDir would be the temporary test directory from the outer test
3. all calls to `file(...)` below would need to be `rootDir.file(...)`. This avoids the string concatenation problem.
4. There might be other changes that I haven't noticed